### PR TITLE
Pyjs 0 18

### PR DESCRIPTION
**Pull request by DerThorsten**

### CODE REVIEW

File: environment-wasm-host.yml [hunk 1]
```diff
@@ -14,5 +14,9 @@ dependencies:
...
   - xeus 
   - xeus-python-shell>=0.5
-  - pyjs>=0.7.0,<0.8
-  - wasm_requests>=0.1.5
+  - pyjs>=0.18.2
+  - requests-wasm-polyfill>=0.1.5
+  - bzip2 
+  - sqlite 
+  - zlib 
+  - libffi
```
Comment (JohanMabille) [inline environment-wasm-host.yml:22]: Why do we need these additional dependencies?

**Comment by DerThorsten** [comment]
we need this whenever we *link* against lib-python.
But we cannot / should-not make this a run dependency of python. Otherwise all side-packages will have these libraries as dependencies

**Comment by DerThorsten** [comment]
@JohanMabille we can use `libpython` instead of `python` https://beta.mamba.pm/channels/emscripten-forge/packages/libpython which has these additional pkgs as dependencies

**Comment by JohanMabille** [comment]
Ok, thanks for the explanation. And +1 for the meta package!

**Pull request merged**